README.md

```diff
@@ -29,11 +29,12 @@ To add a new brand:
 1. add icons, strings etc. in `src/<new_brand>`
 1. to enable automated deployments, add the `new_brand` to `.travis.yml`
 
-# Publishing
+# Releasing
 
-Create a git tag starting with `v`, e.g. `v1.2.3` and push the tag to GitHub. 
-
-Creating this tag will trigger a Travis CI to build, sign, and properly version the build. The release-ready APKs are available for side-loading from [GitHub Releases](https://github.com/medic/medic-android/releases) and are uploaded to the Google Play Console in the "alpha" channel. To release to the public, click "Release to Production" or "Release to Beta" via the Google Play Console for each flavor.
+1. Create a git tag starting with `v`, e.g. `v1.2.3` and push the tag to GitHub. 
+2. Creating this tag will trigger a Travis CI to build, sign, and properly version the build. The release-ready APKs are available for side-loading from [GitHub Releases](https://github.com/medic/medic-android/releases) and are uploaded to the Google Play Console in the "alpha" channel for each flavor.
+3. Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org), under the "Product - Releases" category.
+4. When a project is ready to update their flavor the project manager will click "Release to Production" or "Release to Beta" via the Google Play Console.
```

Comment:
"project manager" seems overly specific. The team using this release process could potentially be just one person.

Reply:
So you're saying that devs do step 4 as well?

Reply:
I don't think we want devs automatically publishing specific flavours, because there may be dependencies on cht-core being upgraded. This step is akin to someone triggering a horti upgrade for a instance.

Do we have a better title for this role? "Upgrader", "Maintainer", etc?

Reply:
Likely not devs, no. But none of the other steps explicitly have an actor. So maybe all steps should have an actor or none? Maybe this is overly abstract, but I think external contributors shouldn't need to know how medic is organised to understand these steps.

Maybe just "Each flavor is then individually released to users via "Release Management" in the Google Play Console. Once a flavor has been tested and is ready to go live, click _Release to Production_"?

Reply:
Done.